python/demo/demo_lagrange_variants.py

```diff
@@ -24,9 +24,8 @@
 
 import basix
 import basix.ufl
-import ufl  # type: ignore
+import ufl
```

Comment:
seems to me that here the other convention is used

Reply:
Indeed